src/sentry/api/serializers/models/organization.py

```diff
@@ -243,6 +244,7 @@ def serialize(
             "dateCreated": obj.date_added,
             "isEarlyAdopter": bool(obj.flags.early_adopter),
             "organizationUrl": generate_organization_url(obj.slug),
+            "regionUrl": generate_region_url(),
```

Comment:
Do we have enough URLs now that we should have a nested structure? Something like

```json
{
  "links": {
    "regionApi": "https://us.sentry.io",
    "organization": "https://acme.sentry.io",
    "sentry": "https://sentry.io"
  }
}
```